AI Generative article (#1995)
Codecademy/docs: content/ai/concepts/generative-ai/generative-ai.md

### Speed of change

What is different about Generative AI is the speed of change. This is creating fear of the unknown as it is difficult to predict what will happen next. Digital artists and some tech leaders such as Elon Musk and Steve Wozniak are calling for the halting of Generative AI until the impacts on society are more understood.

### How we work

Generative AI is expected to have significant impacts on the working world. Directly hundreds of millions of jobs will be affected.

In some sense the change is expected to be welcome as jobs that are currently mundane or have been difficult to automate will go away. For example frame editing, common artwork generation, and basic copywriting.

One concern with this reduction is how jobs typically mentor juniors. If the mundane entry work goes away - how will the next generation be trained?

### Possibilities

While any new disruptive technology creates a lot of uncertainties many people are also excited about what the future may bring.

> Generative AI may open up entirely new ways we can create and consume content. For example media now is difficult and time consuming to create even just in one length and format. Generative AI may allow different versions of movies, stories, and short videos to be generated from the same source media to allow new ways of media consumption.

Reviewer: ```suggestion
Generative AI may open up entirely new ways to create and consume content. For example, currently, any form of media is difficult and time-consuming to create. Generative AI may allow different versions of movies, stories, and short videos to be generated from the same source to allow new ways of media consumption.
```

> Generation of speech based on text with high fidelity style will allow creates to make content where they were limited by voice overlay before.

Reviewer: ```suggestion
The generation of high-fidelity speech based on text will allow creators to make content when previously, they were limited by voice overlay.
```

> Code generation, while automating repetitive work may result in higher quality code and the focus on new creative user centric solutions.

Reviewer: ```suggestion
Code generation, while automating repetitive work, may result in higher-quality code and allow engineers to focus on creative user-centric solutions.
```

> Movies and video creation once expensive and available through large studios and digital production groups can instead be created by individuals and bound only by their imagination.

Reviewer: ```suggestion
Movies and video creation, once expensive and available only through large studios and digital production groups, can instead be created by individuals and bound only by their imagination.
```

Content may be connected back through the block chain to compensate original creators.

> Generative AI may help people understand more how creative processes work in general and how to continually enhance the human process.

Reviewer: ```suggestion
Generative AI can help people further understand how the creative process works in general and how to continually enhance the human process.
```

> Generative AI may help people understand more how creative processes work in general and how to continually enhance the human process.

Reviewer: "the human process" is unclear. Can a different example be used?
